[2023-10-05T20:10:50.627+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.create_table manual__2023-10-05T20:10:49.482404+00:00 [queued]>
[2023-10-05T20:10:50.661+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.create_table manual__2023-10-05T20:10:49.482404+00:00 [queued]>
[2023-10-05T20:10:50.662+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-10-05T20:10:50.663+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2023-10-05T20:10:50.663+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-10-05T20:10:50.678+0000] {taskinstance.py:1383} INFO - Executing <Task(PostgresOperator): create_table> on 2023-10-05 20:10:49.482404+00:00
[2023-10-05T20:10:50.684+0000] {standard_task_runner.py:55} INFO - Started process 1626 to run task
[2023-10-05T20:10:50.689+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'create_table', 'manual__2023-10-05T20:10:49.482404+00:00', '--job-id', '40', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmp1wxl978a']
[2023-10-05T20:10:50.692+0000] {standard_task_runner.py:83} INFO - Job 40: Subtask create_table
[2023-10-05T20:10:50.770+0000] {task_command.py:376} INFO - Running <TaskInstance: user_processing.create_table manual__2023-10-05T20:10:49.482404+00:00 [running]> on host ca408aed2033
[2023-10-05T20:10:50.847+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=create_table
AIRFLOW_CTX_EXECUTION_DATE=2023-10-05T20:10:49.482404+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-10-05T20:10:49.482404+00:00
[2023-10-05T20:10:50.858+0000] {base.py:71} INFO - Using connection ID 'postgres' for task execution.
[2023-10-05T20:10:50.862+0000] {sql.py:315} INFO - Running statement: 
            CREATE TABLE IF NOT EXISTS users (
                firstname TEXT NOT NULL,
                lastname TEXT NOT NULL,
                country TEXT NOT NULL,
                username TEXT NOT NULL,
                password TEXT NOT NULL,
                email TEXT NOT NULL
            );
            , parameters: None
[2023-10-05T20:10:50.864+0000] {postgres.py:96} INFO - NOTICE:  relation "users" already exists, skipping

[2023-10-05T20:10:50.874+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=create_table, execution_date=20231005T201049, start_date=20231005T201050, end_date=20231005T201050
[2023-10-05T20:10:50.901+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-10-05T20:10:50.938+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
